snapcraft/main.py

```diff
@@ -48,7 +48,8 @@
   -h --help                             show this help message and exit
   -v --version                          show program version and exit
   -d --debug                            print debug information while executing
-                                        (including backtraces)
+                                        (including backtraces). When calling
+                                        cleanbuild opens a shell on failures
```

Comment:
I would be a little more verbose. When used with cleanbuild, it opens a shell in case of failure.